cve_bin_tool/csv2cve.py

```diff
@@ -87,17 +95,30 @@ def generate_modules(self):
                             f'Found CVES for {row["vendor"]} {row["product"]}, version {row["version"]}'
                         )
                         # if we found CVES add to the all_cves
-                        all_cves[row["product"]][row["version"]] = cves
+                        cve_data = dict()
+                        cve_data["vendor"] = row["vendor"]
+                        cve_data["product"] = row["product"]
+                        cve_data["version"] = row["version"]
+                        cve_data["cves"] = cves
+                        cve_data["cve_count"] = len(cve_data["cves"])
```

Comment:
Is there any reason for separately keeping variable for cve_count? why don't just find len(cve_data["cves"]) whenever we need it?

Reply:
yes! we can do that. I'll fix this.